docusaurus.config.js

```diff
@@ -146,28 +146,90 @@ const config = {
         ],
       },
       footer: {
-        style: 'dark',
         links: [
           {
-            title: 'Support',
+            title: 'General',
             items: [
               {
-                label: 'Discord',
-                href: 'https://oasis.io/discord',
+                label: 'Explorer',
+                href: 'https://explorer.oasis.io',
+              },
+              {
+                label: 'Oasis Wallet',
+                href: 'https://wallet.oasis.io',
+              },
+              {
+                label: 'Oasis CLI',
+                href: 'https://github.com/oasisprotocol/cli',
+              },
+              {
+                label: 'Status page',
+                href: 'https://status.oasis.io',
+              },
+            ],
+          },
+          {
+            title: 'dApp',
+            items: [
+              {
+                label: 'Testnet Faucet',
+                href: 'https://faucet.testnet.oasis.dev/',
+              },
+              {
+                label: 'Playground',
+                href: 'https://playground.oasis.io/',
+              },
+              {
+                label: 'API Reference',
+                href: 'https://api.docs.oasis.io/',
+              },
+            ],
+          },
+          {
+            title: 'Node',
+            items: [
+              {
+                label: 'Mainnet',
+                to: 'node/mainnet/',
+              },
+              {
+                label: 'Testnet',
+                to: 'node/testnet/',
               },
             ],
           },
           {
-            title: 'Watch us',
+            title: 'Community',
             items: [
+              {
+                label: 'Discord',
+                href: 'https://oasis.io/discord',
+              },
               {
                 label: 'Youtube',
                 href: 'https://www.youtube.com/channel/UC35UFPcZ2F1wjPxhPrSsESQ',
               },
+              {
+                label: 'Rosegarden',
```

Comment:
This is a community-run project, I'm not sure we should link it in the footer.

Reply:
Replaced with X.

Reply:
I replaced it with another GitHub link.